optuna/visualization/_edf.py

```diff
@@ -29,8 +29,8 @@ class _EDFLineInfo(NamedTuple):
 
 
 class _EDFInfo(NamedTuple):
-    lines: List[_EDFLineInfo]
-    x_values: np.ndarray
+    lines: List[_EDFLineInfo] = []
+    x_values: np.ndarray = np.array([])
```

Comment:
Not sure but I'm wondering if there is a problem to set mutable object in namedtuple.

Reply:
Thanks. Indeed, it has the potential to induce a bug when we modify the attribute of the named tuple and reuse it. We don't modify the attribute after creating this info in the current code, so I believe that it doesn't matter for now. But we agree to use immutable objectives is much safer to avoid the unintended bugs.

Reply:
Agree. We may use dataclass or avoid using namedtuple's default value instead.